solidity/deploy/00_deploy_light_relay.ts

```diff
@@ -0,0 +1,32 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
```

Comment:
I know it will be painful, but we should rename this file prefix to `01_` and bump all the subsequent file's prefixes 😢

Reply:
We should! `00_` should be reserved for *resolve*s. It wasn't that painful, done in 70c01c27ac3799c8b649021ff7b09b638a42a192.